tests/functional/resources/modules/d8/woot/woot.deploy.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Successful deploy hook.
+ */
+function woot_deploy_a()
+{
+    // Note that this is called 'a' so that it will run first. The deploy hooks
+    // are executed in alphabetical order.
+    return t('This is the update message from woot_deploy_a');
+}
+
+/**
+ * Successful batched deploy hook.
+ */
+function woot_deploy_batch(array &$sandbox)
+{
+    module_load_install('woot');
+    return woot_update_8105($sandbox);
```

Comment:
This is a pretty weird example - running an update function. Any chance we use other code?

Reply:
yes sure, I just didn't want to copy the code but of course we can.